src/support/z_sanitize.erl

```diff
@@ -296,6 +296,7 @@ preferred_protocol(<<"www.youtube.", _/binary>>) -> <<>>;
 preferred_protocol(<<"vimeo.", _/binary>>) -> <<>>;
 preferred_protocol(<<"www.vimeo.", _/binary>>) -> <<>>;
 preferred_protocol(<<"platform.instagram.com", _/binary>>) -> <<>>;
+preferred_protocol(<<"platform.twitter.com", _/binary>>) -> <<>>;
```

Comment:
As we fetch javascript here, maybe we should fix this to https?

And then also add that as a preferred protocol to more services...